src/app/compose/draftdesk.service.ts

```diff
@@ -261,16 +261,14 @@ export class DraftDeskService {
         const draftObj = DraftFormModel.create(
             -1,
             this.froms[0],
-            'support@runbox.com',
+            '"Runbox Support" <support@runbox.com>',
             'Runbox 7 bug report'
         );
         const template = await this.http.get('assets/templates/bug_report.txt',
                                              {responseType: 'text'}).toPromise();
         const me = await this.rmmapi.me.toPromise();
 
-        let body = `
-${template}
-`
+        let body = `${template}`
```

Comment:
This could probably just be `let body = template` at this point :)

Reply:
I'm sure there was a point to the newlines, tho I've now forgotten why, I think I originally planned to use variables for the values in the template and interpolate them this way too - but didn't. If newlines not required then yes, just use the template var instead ;)

Reply:
The newlines added newlines in the template too, which caused the text to appear on line 2 in the message. Removing them fixed that issue while not seeming to introduce any new ones. :)